.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -49,37 +49,34 @@
-      - [Internal] [2.1] When running smoke-test locally, also pass `/p:ProdConBlobFeedUrlPrefix=<...>`
-        - The smoke-test command looks like `./build.sh /t:RunSmokeTest /p:Configuration=Release /p:ProdConBlobFeedUrlPrefix=***`.
-      - Update baseline from the CI job, using the `centos71 Offline` artifacts.
-        - `Logs centos71 Offline/source-build/logs/artifacts/prebuilt-report/generated-new-baseline.xml`
-          - => `tools-local/prebuilt-baseline-online.xml`
-        - `Logs centos71 Offline/tarball/logs/artifacts/prebuilt-report/generated-new-baseline.xml`
-          - => `tools-local/prebuilt-baseline-offline.xml`
+      - [Internal] [3.1] Also pass arg `/p:AzDoPat=$internalPackageFeedPat` to build.sh.
+      - [Internal] To run CI, from [dnceng/internal/source-build-pre-arcade-CI](https://dev.azure.com/dnceng/internal/_build?definitionId=251), click "Run pipeline" and select your dev/… branch under "Branch/tag", then click "Run"
+         - If you are unsure if your build will be successful, you can run only the centos71_* stages to save resources.
 1.  - [ ] Review prebuilt baseline diff and iterate builds to drive to zero.
       - Find an explanation for each change in each (online/offline) baseline.
         - If possible, figure out the cause and fix.
           - Discuss with repo or code owner if the fix is a new patch.
           - This may require adding packages to dotnet/source-build-reference-packages.
         - If the baseline change is due to the change in patch version, make sure there's an issue tracking fixing the instability, or start tracking it.
         - If in doubt, talk to the team. Someone might have already encountered a similar diff in the past.
+1.  - [ ] Add team members as reviewers to the PR. Get acceptance.
+      - Note: The PR is internal for internal builds and on GitHub for non-internal builds.
+      - [Non-Internal] Merge the PR when green and accepted.
+      - [Internal] Don't merge the PR.
+1.  - [ ] [Internal] Download the artifacts from the centos71_Online_BaseTarball PR validation build onto a machine with Bash.
+      - From the "Artifacts" page, navigate to "Tarball centos71_Online_BaseTarball" and download the `base_tarball_*.tar.gz` and `tarball_*-smoke-test-prereqs.tar.gz` files.
+      - The smoke-test prereqs are platform-independent but should be taken from this non-portable build, *not* CentOS7 Offline Portable.
+      - Can use <https://github.com/dotnet/source-build/blob/release/3.1/scripts/fetch-azdo-artifact-tarball.sh> as a library.
+        - E.g. download the tarballs with `download_tarball`, and fix them up with `fix_azdo_tarball` if necessary.
+1.  - [ ] [Internal] Rename tarball and smoke-test prereqs to human-friendly names. (Can use `rename_tarball_inner_dir`.)
+      - Refer to previous release's email threads for naming pattern.
+1.  - [ ] [Internal] Upload tarballs and smoke-test prereqs to blob storage.
```

Comment:
What blob storage?  Should a source-build team member do this to control access?

Reply:
I'll add the details for where and where to get the creds anyway, then we can decide whether we want to do it ourselves.  It'll still be useful for new team members.